File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    for string in quiz:
        s = string.split()
        if s[1] =='+':
            if int(s[0]) + int(s[2]) == int(s[4]):
                answer.append("O")
            else:
                answer.append("X")
        if s[1] == '-':
            if int(s[0]) - int(s[2]) == int(s[4]):
                answer.append("O")
            else:
                answer.append("X")
    return answer

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        for (int i = 0; i < quiz.length; i++) {
            String[] c = quiz[i].split(" ");
            int num1 = Integer.parseInt(c[0]);
            int num2 = Integer.parseInt(c[2]);
            int expected = Integer.parseInt(c[4]);
            if (c[1].equals("+")) {
                if (num1 + num2 == expected) {
                    answer[i] = "O";
                } else {answer[i] = "X";}
            } else {
                if (num1 - num2 == expected) {
                    answer[i] = "O";
                } else {answer[i] = "X";}
            }
        }
        return answer;
    }
}